### PR TITLE
Get Webinars from the backend API

### PR DESCRIPTION
**Pull request by SasinduNanayakkara**

### CODE REVIEW

File: src/pages/event/past-webina.tsx [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+import { getWebinars } from '../../api/WebinarAction';
 
 const PastWebinars: React.FC = () => {
+
+  const [webinarList, setWebinarList] = useState<IWebinar>();
+
+  //fetch all webinar items
+  useEffect(() =>{
+    getWebinars()
+    .then((data) =>{
```
Comment (senuravihanjayadeva) [inline src/pages/event/past-webina.tsx:16]: change "data" to "response"

File: src/pages/event/past-webina.tsx [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+      setWebinarList(data.data);
```
Comment (senuravihanjayadeva) [inline src/pages/event/past-webina.tsx:17]: response.data

File: src/pages/event/past-webina.tsx [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
 import { CARD_TYPE_WEBINA, SLIDER_RESPONSIVE_BREAKPOINTS } from '../../constants';
 import Slider from 'react-owl-carousel';
-import webinars from '../../data/PastWebinarsData.json';
 import {translation} from '../../locales/en-US/translation.json';
+import { IWebinar } from '../../interfaces/IWebinar';
+import { getWebinars } from '../../api/WebinarAction';
 
 const PastWebinars: React.FC = () => {
+
+  const [webinarList, setWebinarList] = useState<IWebinar>();
```
Comment (senuravihanjayadeva) [inline src/pages/event/past-webina.tsx:11]: const [webinarList, setWebinarList] = useState<IWebinar[]>([]);

File: src/pages/event/past-webina.tsx [hunk 4]
```diff
@@ -58,7 +73,7 @@ const PastWebinars: React.FC = () => {
         </div>
       </div>
 
-      {webinars && webinars.data.length > 0 && 
+      {webinarList && webinarList.Items.length > 0 &&
```
Comment (senuravihanjayadeva) [inline src/pages/event/past-webina.tsx:76]: webinarList.Items ?

File: src/pages/event/past-webina.tsx [hunk 5]
```diff
@@ -70,7 +85,7 @@ const PastWebinars: React.FC = () => {
             slider = slide;
           }}
         >
-          {webinars.data.map((event, index) => (
+          {webinarList.Items.map((event, index) => (
```
Comment (senuravihanjayadeva) [inline src/pages/event/past-webina.tsx:88]: webinarList.Items ?

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Review by senuravihanjayadeva — APPROVED** [review]

**Review by miyurugunarathna — APPROVED** [review]
LGTM

**Pull request merged**